src/test/java/com/smartcar/sdk/AuthClientTest.java

```diff
@@ -193,12 +193,32 @@ public void testAuthUrlBuilderDefault() throws Exception {
   }
 
   @Test
-  public void testAuthUrlBuilderWithOptions() throws Exception {
```

Comment:
I think you should leave this test as-is and just add a new test for simulated mode. The new test doesn't need to set things like `state`, `makeBypass`, etc.